src/config/dynamic-theme-fixes.config

```diff
@@ -216,6 +216,19 @@ div.description > span {
 
 ================================
 
+500px.com
+
+CSS
+.button.new_fav .icon .shape {
+    fill: transparent !important;
+    stroke: rgb(232, 230, 227);
+}
+.button.only_icon.hearted .icon .shape, .button.only_icon.liked .icon .shape {
+    fill: var(--darkreader-inline-fill) !important;
+}
```

Comment:
```suggestion

IGNORE INLINE STYLE
.button.only_icon .icon .shape
```

Reply:
Hi @Gusted, sorry I'm late.
I have tested your solution and it is better than mine.
I will remember it for the next pull requests!